.gitignore

```diff
@@ -0,0 +1,2 @@
+/env/
+/.ipynb_checkpoints/
```

Comment:
[critical] 

1. Looks like `/.ipynb_checkpoints/` does not work. I see this changes in git diff.
2. Please add `.idea/` to gitignore too.

Reply:
/.ipynb_checkpoints/ was added to my gitignore in error